test-integ/peering_commontopo/ac5_2_pq_failover_test.go

```diff
@@ -86,7 +86,10 @@ func (s *ac5_2PQFailoverSuite) setupDC(ct *commonTopo, clu, peerClu *topology.Cl
 		Service: NewFortioServiceWithDefaults(
 			clu.Datacenter,
 			serverSID,
-			nil,
```

Comment:
@NiniOak I'm not sure why these two services had service mesh enabled? Assuming it was just an oversight, so I switched it.